### DISCUSSION TITLE
Automatic categorisation of expenses

### DISCUSSION THREAD

**Discussion by tonypius** [ananthakumaran/paisa#205] [Ideas]
Is there any way to use an LLM like chatGPT to auto classify transactions to relevant categories. Also, if i've to go about building this feature now , how would i go about it ?

**Comment by ananthakumaran** [comment]
Paisa itself doesn't do it as of today, but the ledger file is just a plain text file you can do it via other means. There is one discussion about using LLM on ledger-cli mailing list, you can check there to get started.

**Reply by SagnikPradhan** [reply]
Do you have a link to it? Having trouble finding it
